docs/installation.rst

```diff
@@ -6,60 +6,80 @@ General
-* qcli
-* openssl
-* rsync
-* gcp (installed via gnu-coreutils on OSX)
-* rawcooked
 * 7zip aka 7za aka p7zip-full
+* asdcplib (https://github.com/cinecert/asdcplib, required by clairmeta for checking DCP)
+* exiftool (https://exiftool.org/install.html)
 * md5deep
 * mediaconch
+* qcli
+* rawcooked
+* rsync
+* siegfried aka sf (https://www.itforarchivists.com/siegfried)
+* sox (SoundeXchange, https://sourceforge.net/projects/sox/, required by clairmeta)
+
+Some dependencies below are not our common dependencies but could be needed for specific purpose:
+
+* gcp (copy from/to LTO, installed via gnu-coreutils on OSX)
+* git (for update.py, no longer in use for updating scripts)
+* mkvpropedit (for concat.py, installed via mkvtoolnix)
+* openssl
 
 Specific Instructions - Windows
 -------------------------------
 
 * install 64-bit git-bash using all the default settings https://git-scm.com/downloads - make sure it's the 64-bit version!
 * install 64-bit python3, making sure to tick the option to ADD TO PATH https://www.python.org/downloads/
-* open cmd and ``mkdir ifiscripts`` and ``git clone https://github.com/kieranjol/ifiscripts``
-* add this ifiscripts path  (eg ``C:\Users\KAZETCCCC\ifigit\ifiscripts``)to the environmental path, following these steps: https://www.computerhope.com/issues/ch000549.htm
+* open cmd and ``mkdir ifiscripts`` and ``git clone https://github.com/Irish-Film-Institute/IFIscripts.git``
+* add this ifiscripts path  (eg ``C:\Users\$USER\ifigit\ifiscripts``)to the environmental path, following these steps: https://www.computerhope.com/issues/ch000549.htm
 * ffmpeg the default option works well - 64-bit static https://ffmpeg.zeranoe.com/builds/ and place in scripts folder
   * OR install media-autobuild-suite but extract to the C:\mas folder due to long path issues
 * mediainfo - get the 64-bit CLI version https://mediaarea.net/en/MediaInfo/Download/Windows
-* install lxml with ``pip install lxml``
+* install lxml with ``pip install lxml`` if not installing ifiscripts by pip
 * install siegfried exe (https://www.itforarchivists.com/siegfried/)file to the ifiscripts folder and run ``sf -update`` in cmd 
-* download exiftoolinstaller and select the 'latest build' option - make sure that the option to add to path is ticked - https://oliverbetz.de/pages/Artikel/ExifTool-for-Windows
-* install notepad++  - https://notepad-plus-plus.org/downloads/
-* install libreoffice - https://www.libreoffice.org/download/download/
+* download exiftool installer and select the 'latest build' option - make sure that the option to add to path is ticked - https://oliverbetz.de/pages/Artikel/ExifTool-for-Windows
+* install notepad++ - https://notepad-plus-plus.org/downloads/
+* install libreoffice for accessing and editing csv files - https://www.libreoffice.org/download/download/
 
 
 Specific Instructions - Ubuntu
 ---------------------
+
+(We no longer support Ubuntu but the scripts are still working)
```

Comment:
Is it not needed for some copyit workflows for linux DCP drives?

Reply:
We now request NTFS drives as most of IFA workstations are now Windows. We have a macbook for OA workflow if needed. And sometimes we received Linux drives and we just need wsl to copy it to a workstation.